src/help.cpp

```diff
@@ -251,6 +251,14 @@ void add_options() {
   opt->help = "Number of initial records in table";
```

Comment:
Please change this:
opt->help = "initial records in table are inserted randomly in the range of 0 to N";